system-tests/dashboard/test-dashboard.js

```diff
@@ -0,0 +1,394 @@
+require("../_support/utils/ServicesUtil");
+const { Timeouts } = require("../_support/constants");
+
+describe("Dashboard", function() {
+  beforeEach(() => {
+    cy.visitUrl("dashboard");
+  });
+
+  afterEach(() => {
+    cy.window().then(win => {
+      win.location.href = "about:blank";
+    });
+  });
+
+  function deleteService(serviceName) {
+    // Delete the service
+    cy.visitUrl(`services/overview/%2F${Cypress.env("TEST_UUID")}`);
+
+    // Click on the serviceName of the package to delete
+    cy
+      .get(".page-body-content table")
+      .getTableRowThatContains(serviceName)
+      .get("a.table-cell-link-primary")
+      .contains(serviceName)
+      .click();
+
+    // Click delete in the dropdown
+    cy
+      .get(".page-header-actions .dropdown")
+      .click()
+      .get(".dropdown-menu-items")
+      .contains("Delete")
+      .click();
+
+    // Confirm the deletion
+    cy.get(".modal.confirm-modal input").type(serviceName);
+    cy.get(".modal.confirm-modal button.button-danger").click();
+
+    cy
+      .get(".page-body-content table")
+      .contains(serviceName)
+      .should("not.exist");
+  }
+
+  function createService(serviceName) {
+    // First run a service
+    cy.visitUrl(`services/overview/%2F${Cypress.env("TEST_UUID")}/create`);
+
+    const cmdline = "while true; do echo 'test' ; sleep 100 ; done";
```

Comment:
not big deal but I would suggest to have this on the top just for the matter of consistency.